### DISCUSSION TITLE
suddenly getting  xml parsing error occurred trying to get md5 data

### DISCUSSION THREAD

**Discussion by ohjays** [Kalanyr/gogrepo#56] [General]
probably due to a debian update, any idea ?

output as follows
18:25:54 | loading local manifest...
18:25:54 | fetching game product data (page 1)...
18:25:55 | fetching game product data (page 2 / 2)...
18:25:56 | found 191 games !!!
18:25:56 | (  1 / 191) fetching game details for 1nsane...
18:26:02 | xml parsing error occurred trying to get md5 data for setup_1nsane_1.0_(18948).exe
18:26:10 | (  2 / 191) fetching game details for absolute_drift...
18:26:17 | xml parsing error occurred trying to get md5 data for setup_absolute_drift_1.0g_(64bit)_(49142).exe
18:26:17 | (  3 / 191) fetching game details for act_of_war_high_treason...
18:26:24 | xml parsing error occurred trying to get md5 data for setup_act_of_war_direct_action_1.06.3_(24183).exe

**Comment by Kalanyr** [comment]
It's a known issue, GOG is changing the way they provide md5 data.  I'm
giving it time, to be sure what the new method is and will probably work on
a patch over the weekend.
